open-company/open-company-web
# Improve mobile edit

Design: https://projects.invisionapp.com/share/VAFLD1KR9BG#/screens

To test:
- on your mobile device
- tap new post
- tap Post (w/o title and body)
- [x] do you see the tooltip suggesting to add at least a title? Good
- add a title
- Save as draft
- [x] are you redirected to the Drafts board? Good
- [x] do you see the new draft? Good
- select the draft
- add a body
- post it
- [x] are you redirected to the board? Good
- [x] do you see the new post? Good
- tap new Post again
- add a title and a body
- tap on the board name
- switch board
- Post
- [x] are you redirected to the correct board? Good
- [x] do you see the new post? Good
- tap on the vertical ellpsis of the FoC
- tap on edit
- edit the post (best the title so it's visible from the FoC)
- Save
- [x] do you see the changes from the board? Good
- merge
